webapp/portlet/src/main/webapp/vue-apps/attach-image/components/form/ImageInputMultiUpload.vue

```diff
@@ -132,6 +134,16 @@ export default {
       const now = Date.now();
       return `${random}-${now}`;
     },
+    handlePasteFiles(event) {
+      const files = event?.clipboardData?.files || event?.clipboardData?.items;
+      if (files?.length) {
+        const files = event.clipboardData.files;
```

Comment:
```suggestion
      const files = event?.clipboardData?.files || event?.clipboardData?.items;
      const files = event.clipboardData.files;
```
two `const files`

Reply:
Besides have you tested with `event?.clipboardData?.items` only ?

Reply:
yep and didn't working!

Reply:
duplicated files const removed!

Reply:
> yep and didn't working!

It works in fact, having both may be better.

![image](https://github.com/Meeds-io/social/assets/594824/d1a538c7-fd11-4494-9f3d-1b2956e3c0ba)

Besides, the CKEditor `paste` event wasn't handled, I will help you providing it.

Reply:
> Besides, the CKEditor `paste` event wasn't handled, I will help you providing it.

I fixed it using commit https://github.com/Meeds-io/social/commit/9418104774dfa4d757ccf45a9b34fb93a4a0690c

This will avoid to upload files when using `Ctrl + V` when editor isn't ready nor displayed.